src/feat/feature-fbank.h

```diff
@@ -55,7 +56,8 @@ struct FbankOptions {
                  energy_floor(0.0),  // not in log scale: a small value e.g. 1.0e-10
                  raw_energy(true),
                  htk_compat(false),
-                 use_log_fbank(true) {}
+    use_log_fbank(true),
```

Comment:
there is odd indentation here.  Please make sure you are writing spaces not tabs.